### PR TITLE
License file location in the summary.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

**Comment by kevmoo** [comment]
Format!

File: lib/src/license.dart [hunk 1]
```diff
@@ -53,22 +55,31 @@ abstract class LicenseNames {
...
-    return [];
-  }
-  return [await detectLicenseInFile(licenseFile)];
+Future<List<LicenseFile>> detectLicensesInDir(String baseDir) async {
+  final rootFiles = await new Directory(baseDir).list().toList();
+  final licenseCandidates =
+      rootFiles.where((fse) => fse is File).where(_isLicenseFile).toList();
+  if (licenseCandidates.isEmpty) return [];
+
+  final licenses = await Future.wait(licenseCandidates.map(
```
Comment (kevmoo) [inline lib/src/license.dart]: Assuming this list is not dozens long, I guess it's fine...

Comment (isoos) [reply lib/src/license.dart]: shall we limit it to 10?

Comment (kevmoo) [reply lib/src/license.dart]: Something like that...

**Pull request merged**